akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSupervisionSpec.scala

```diff
@@ -288,49 +284,6 @@ class InterpreterSupervisionSpec extends AkkaSpec with GraphInterpreterSpecKit {
       }
     }
 
-    "resume when Scan throws" in new OneBoundedSetup[Int](Seq(
```

Comment:
Restart is covered by existing test "fail properly" (great name btw), but  not resume?